soh/src/code/z_message_PAL.c

```diff
@@ -2609,7 +2609,7 @@ void Message_DrawMain(PlayState* play, Gfx** p) {
                     if (msgCtx->lastPlayedSong < OCARINA_SONG_SARIAS &&
                         (msgCtx->ocarinaAction < OCARINA_ACTION_PLAYBACK_MINUET ||
                          msgCtx->ocarinaAction >= OCARINA_ACTION_PLAYBACK_SARIA)) {
-                        if (msgCtx->disableWarpSongs || (interfaceCtx->restrictions.warpSongs == 3 && !IS_RANDO)) {
```

Comment:
interfaceCtx->restrictions.warpSongs is interesting since, unlike other restriction fields that are either 1 or 0, some scenes have it set to 0, 1, or 3. However, it is only set to 1 in scenes that do not also restrict ocarina. I suppose this was to restrict certain songs in certain contexts. However, the only scenes like this are ones where the play does not have control, like the chamber of the sages. So, there is no actual use for this field to be anything other than 0 or 3. For SceneDB, I have simplified this to be a bool.